docs/configuration.rst

```diff
@@ -17,11 +17,22 @@ For other use cases, you can customize these configuration options in your ``con
 
    Context passed to the template defined by ``notfound_template``.
 
-   Default: ``{'title': 'Page not found', 'body': '<h1>Page not found</h1>\n\nThanks for trying.'}``
+   Default:
+
+   .. code-block:: python
+
+      {
+          'title': 'Page not found',
+          'body': '<h1>Page not found</h1>\n\nThanks for trying.',
```

Comment:
This reads a bit passive aggressive, we should probably replace `Thanks for trying` with something different, or just drop it.

Reply:
Yeah. I want to change this eventually to a nicer text and mention the sidebar as well.

Reply:
This PR only touched this because it changes the style, but the content is old.